Networking/Networking/Model/OrderItem.swift

```diff
@@ -52,17 +55,36 @@ public struct OrderItem: Decodable, Hashable {
         self.taxes = taxes
         self.total = total
         self.totalTax = totalTax
+        self.attributes = attributes
     }
 
     /// The public initializer for OrderItem.
     ///
     public init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         let itemID = try container.decode(Int64.self, forKey: .itemID)
-        let name = try container.decode(String.self, forKey: .name).strippedHTML
         let productID = try container.decode(Int64.self, forKey: .productID)
         let variationID = try container.decode(Int64.self, forKey: .variationID)
 
+        let isVariation = variationID > 0
+        let name: String
+        let attributes: [OrderItemAttribute]
+        if isVariation {
+            name = try ((container.decodeIfPresent(String.self, forKey: .variationParentName))
+                        ?? container.decode(String.self, forKey: .name)).strippedHTML
+            let allAttributes = (try? container.decodeIfPresent([OrderItemAttribute].self, forKey: .attributes)) ?? []
+            attributes = allAttributes.filter { $0.name != "_reduced_stock" }
+
+            // Logs error for unexpected attributes.
+            let unexpectedAttributes = attributes.filter { $0.name.hasPrefix("_") }
+            if unexpectedAttributes.isEmpty == false {
+                DDLogError("⚠️ Unexpected order item attributes: \(unexpectedAttributes)")
+            }
```

Comment:
nit: I think that we can remove the check on `_reduced_stock` and filter by removing just the attributes with an underscore `_`prefix, right?

Reply:
the reason I only filtered `_reduced_stock` is that this is the only attribute added by core that we know of. It's not impossible some users might actually name their attributes with an underscore prefix, so I didn't want to assume for the cases we don't know. It's probably rare though